### PR TITLE
Adding auth0 to personal dash

### PR DESCRIPTION
**Pull request by nisaChampagne**
# Description
Mocked up personal dashboard so that I could add different queried data to differentiate between dashboards. Also set up a private route to check if authenticated through auth0 before access is given to personal or team dashboards. The mocked up personal dashboard is very basic and is titled under TestPersonalDashboard.js
    
    Fixes # (issue)   
    - [x] New feature (non-breaking change which adds functionality)
    
    ## Change status
    - [x] Complete, tested, ready to review and merge
    
    Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
    
    - [x] Test A
    - Tested in brower
    
    # Checklist:
    Everything in this checklist is required for each PR.  Please do not approve a PR that does not have all of these items.  
    
    - [x] I have pulled from master before submitting this PR
    - [x] My code follows the style guidelines of this project
    - [x] I have performed a self-review of my own code
    - [x] I have commented my code, particularly in hard-to-understand areas
    - [ ] I have made corresponding changes to the documentation
    - [x] My changes generate no new warnings
    - [ ] I have added tests that prove my fix is effective or that my feature works
    - [ ] New and existing unit tests pass locally with my changes
    - [x] There are no merge conflicts
    - [ ] Any env variables or config options that are needed have been stated in the README
    - [ ] My code has been reviewed by at least one peer (the peer review to approve a PR counts.  The reviewer must download and test the code)
    - [x] NO API KEYS ARE PUSHED UP

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -87,6 +86,20 @@ functions:
       - http:
           path: /shakespeareQuotes
           method: get
+  gitHubBigQuery:
+    handler: handler.gitHubBigQuery
```
Comment (seanwu20) [inline serverless.yml]: Since the handler isn't in the root folder anymore, this needs to change

File: serverless.yml [hunk 2]
```diff
@@ -87,6 +86,20 @@ functions:
...
+    name: 'github-${self:provider.stage}'
+    events:
+      - http:
+          path: /commits
+          method: get
+  moonPhases:
+    handler: handler.moonPhases
```
Comment (seanwu20) [inline serverless.yml]: Since the handler isn't in the root folder anymore, this needs to change

File: lambda/handler.js [hunk 3]
```diff
@@ -1,4 +1,6 @@
 // Contains test functions right now
+///added headers to avoid CORS issues for now whil in dev
+///added methods for new queries
 
 'use strict';
 const shakespeare = require('./queryShakespeare')
```
Comment (seanwu20) [inline lambda/handler.js]: You've imported shakespeare, but not moon and gitHub

File: lambda/handler.js [hunk 4]
```diff
@@ -1,7 +1,11 @@
...
+const github = require('./queryGitHub')
```
Comment (seanwu20) [inline lambda/handler.js]: You've named this import github, but you used gitHub in the query

Comment (nisaChampagne) [reply lambda/handler.js]: done 👍

**Pull request merged**